### DISCUSSION TITLE
FR: Plugin License Renewals - “Total Bill” Dashboard Widget

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8680] [Ideas]
In the absence of any meaningful/practical way via the Control Panel to get an idea of how much plugin license subscriptions will cost for your CMS instance either individually or in totality I’m putting this dashboard widget suggestion forward.

A simple widget that looks at all the plugins you currently have installed (excluding trial installs) and provides a total $ amount for the annual subscription cost. It would be cool if it included a toggle to switch the $ figure displayed based on all installed plugins vs all installed plugins with “auto renew” enabled. Keeping it simple, the plugin would simply look at all installed plugins and total up their renewal charge. This would give site owners some idea of what their ongoing annual costs would be for CraftCMS, Commerce + paid plugins. 

Right now, via the Control Panel, other than painstakingly searching in the store for a plugin you have installed and then scrolling all the way to the bottom of the plugin detail page there is no other way to see the renewal cost before the plugin expires. This might be fine for one or two plugins but when you have many I’m resorting to a spreadsheet to keep track of them across a number of sites.

**Comment by brandonkelly** [comment]
The best way to do this is from Craft ID, where you can bulk-renew a Craft license along with its plugin licenses all together, even if the licenses originally had different expiration dates.
